packages/flutter_tools/lib/src/base/error_handling_io.dart

```diff
@@ -490,11 +490,12 @@ class ErrorHandlingProcessManager extends ProcessManager {
   }
 }
 
-void _handleLinuxException(Exception e, String message, int errorCode) {
+void _handlePosixException(Exception e, String message, int errorCode) {
   // From:
   // https://github.com/torvalds/linux/blob/master/include/uapi/asm-generic/errno.h
   // https://github.com/torvalds/linux/blob/master/include/uapi/asm-generic/errno-base.h
```

Comment:
nit. is there either POSIX documentation for these, or macos specific docs?

Reply:
done